StoryDomain/Models/Story.cs

```diff
@@ -21,7 +21,9 @@ public class Story : ObjectWithState, IEntity<int> {
 
         public StoryType Type { get; set; }
 
-        public virtual UserInfo User { get; set; }
+        public DateTime? Date { get; set; }
+
+        //public virtual UserInfo User { get; set; }
```

Comment:
Why was this removed.

Reply:
Because I cant save a story, altough I set the User with the method FindUserById, it gives an error.